src/applications/Machine.java

```diff
@@ -47,10 +47,6 @@ public Job getActiveJob() {
         return activeJob;
     }
 
-    public void setActiveJob(Job activeJob) {
```

Comment:
Awesome job with cleaning up the workspace!

Reply:
I found this by looking at the code coverage. The coverage was down from the baseline before this method was removed since it wasn't being called anymore; chasing down exactly which part was untested lead me to check if this method was still used.
Also, I don't believe VScode caught this, but IntelliJ did.

Reply:
Very cool.

When you say "I don't believe VScode caught this, but IntelliJ did", are you saying that IntelliJ noticed that no one was calling it anymore and highlighted that fact, and that VS Code didn't? That would be (minorly) surprising and certainly interesting.

Reply:
@NicMcPhee 
I meant that I did notice IntelliJ presented the code in a way that indicated that it wasn't used, but that I didn't notice anything in VScode, regardless of whether it actually did. (not sure if I was just being oblivious or VScode wasn't set up well)